src/Settings.php

```diff
@@ -425,15 +411,6 @@ public function __construct()
                         'id' => 'google_tag_manager_domain',
                         'type' => 'text',
                     ],
-                    [
-                        'name' => __('AB Hide Selector', 'planet4-master-theme-backend'),
-                        'desc' => __(
-                            'When running an AB test it is possible that the original is shown for a short while (called flicker). If this happens you can enter a CSS selector here and matching elements will be hidden until the test is fully loaded.',
-                            'planet4-master-theme-backend'
-                        ),
-                        'id' => 'ab_hide_selector',
```

Comment:
This one should also be removed from the context, we set it [here](https://github.com/greenpeace/planet4-master-theme/blob/main/src/MasterSite.php#L700)

Reply:
Ah I missed that. It's now removed.